source/includes/auto-encryption-options.rst

```diff
@@ -0,0 +1,128 @@
+The following table describes the structure of an
+``AutoEncryptionOptions`` object:
+
+.. list-table::
+   :header-rows: 1
+   :widths: 30 70
+
+   * - Name
+     - Description
+
+   * - **keyVaultClient**
+     - **Type:** ``MongoClient``
+
+       **Description**
+       A ``MongoClient`` configured to connect to
+       the MongoDB instance hosting your {+key-vault-long+}. For more
+
+       If you omit the ``keyVaultClient`` option, the driver uses the
+       MongoDB instance specified to your ``MongoClient`` containing the
+       ``AutoEncryptionOpts`` configuration as the host of your key vault
+       collection.
+
+       To learn more about {+key-vault-long+}s, see :ref:`qe-reference-key-vault`.
+
+       **Default:** ``undefined``
+       **Accepted Values:** A configured ``MongoClient``
+
+   * - **keyVaultNamespace**
+     - *Required*
+
+       **Type:** String
+
+       **Description:**
+       The full :term:`namespace` of the {+key-vault-long+}.
+
+       **Accepted Values:** A string containing the full namespace of the
+       {+key-vault-long+}.
+
+   * - **kmsProviders**
+     - *Required*
+
+       **Type:** Object
+
+       **Description:**
+       The {+kms-long+} (KMS) used by {+qe+} for
+       managing your {+cmk-long+}s (CMKs).
+
+       To learn more about ``kmsProviders`` objects, see
+       :ref:`qe-fundamentals-kms-providers`.
+
+       To learn more about {+cmk-long+}s, see :ref:`qe-reference-keys-key-vaults`.
+
+       **Accepted Values:** An object containing the KMS provider or providers.
+
+   * - **tlsOptions**
+     -  **Type:** Object
+
+       **Description:**
+       Options to configure ``TLS`` connections to KMS providers.
```

Comment:
Issue: TLS shouldn't be monospace. Applies to all instances in which it's used to describe the mechanism rather than value/parameter/argument/etc.